sql/idempotent/010-processing.sql

```diff
@@ -0,0 +1,85 @@
+
+-------------------------------------------------------------------------------
+-- processing_none
+create or replace function ai.processing_none() returns jsonb
+as $func$
+    select jsonb_build_object
+    ( 'implementation', 'none'
+    , 'config_type', 'processing'
+    )
+$func$ language sql immutable security invoker
+set search_path to pg_catalog, pg_temp
+;
+
+-------------------------------------------------------------------------------
+-- processing_cloud_functions
+create or replace function ai.processing_cloud_functions
+( batch_size int default 50
+, concurrency int default 1
```

Comment:
I would actually default these to NULL. The reason? NULL would mean "default". So if we ever wanted to change the default we could just change in in the lambda. Otherwise no way to know if batch_size of 50 was explicitly set or the default value and thus lambda wouldn't know if it's "free" to change it.

Reply:
good idea. i like that